pkg/obfuscate/sql_tokenizer.go

```diff
@@ -74,6 +74,19 @@ const (
 	TableName
 	ColonCast
 
+	// PostgreSQL specific JSON operators
+	JSONSelect
+	JSONSelectText
+	JSONSelectPath
+	JSONSelectPathText
+
```

Comment:
Can we remove this line and add the specific set of characters that these tokens represent as a comment to the right of each identifier? e.g.
```go
    JSONAnyKeyExists // ?|
```
For all the other ones (non JSON*), it is clear, but here it is less so, so writing a small comment helps.